tuplex/core/src/ee/local/LocalBackend.cc

```diff
@@ -459,12 +459,14 @@ namespace tuplex {
     }
 
     std::vector<IExecutorTask*> LocalBackend::createLoadAndTransformToMemoryTasks(
-            tuplex::TransformStage *tstage, const tuplex::ContextOptions &options,
-            tuplex::codegen::read_block_f functor) {
+            tuplex::TransformStage *tstage,
+            const tuplex::ContextOptions &options,
+            TransformStage::JITSymbols *syms) {
```

Comment:
use shared_ptr

Reply:
The TransformStage pointer comes directly from LocalBackend's `void execute(PhysicalStage* stage) override;` function. Should I turn the regular pointer into a shared pointer to pass it into the `createLoadAndTransformTasks` function?

Reply:
Hm...I think we could that, but that would require to update all the other stages derived from PhysicalStage as well. If you'd like to do that, sure - go ahead.